docs/principles/version.md

```diff
@@ -63,6 +64,10 @@ In increasing order of stability, from least to most stable, these are:
     - During its availability, it must not be revised with incompatible and breaking changes.
     - It may be revised with backwards-compatible, additive changes.
 
+`experimental` is where new API features are incubated. Typically experiments are unlisted in official product documentation, though they may be publicly routed. They're a place to try new things, stress test, invite early feedback, and iterate. Experiments are a temporary draft of an API, so they are considered deprecated upon release, and are considered sunset, expiring 90 days after their release. By this time, an experiment should iterate with a new release version, or promote to `beta`.
```

Comment:
Not sure what's being said about sunset; that they're already deprecated, so in a sunset state, and will be completely expired in 90? Or that they're deprecated at release, and will be sunset/expired in 90 days? Do we have a clear distinction between expired and sunset?

Reply:
I'll whittle that down and just say "sunset after 90 days".